bokehjs/examples/anscombe/anscombe.ts

```diff
@@ -1,8 +1,8 @@
 namespace Anscombe {
   console.log(`Bokeh ${Bokeh.version}`);
-  Bokeh.set_log_level("debug");
+  Bokeh.set_log_level("info");
 
-  const anscombe_quartet = Bokeh.LinAlg.transpose([
+  const anscombe_quartet = Bokeh.Utils.transpose([
```

Comment:
This seems "OK" to me. Whatever we end up with for 0.12.5 should be considered final.